compiler/control/OMROptions.hpp

```diff
@@ -108,7 +108,6 @@ enum TR_CompilationOptions
    TR_RegisterMaps               = 0x00001000,
    TR_CreatePCMaps               = 0x00002000,
    TR_OptimizeForSpace           = 0x00004000,
-   TR_FullSpeedDebug             = 0x00008000,
    TR_MimicInterpreterFrameShape = 0x00008000, // ** NOTE ** Alias for TR_FullSpeedDebug until we're prepared to separate these two concepts
```

Comment:
Can we delete the comment to reflect the fact that there is no more aliasing with your change ?